sound/soc/sof/sof-client-probes.c

```diff
@@ -249,6 +249,28 @@ static int sof_probes_points_remove(struct sof_client_dev *cdev,
 	return ret;
 }
 
+static int sof_probes_get_modules(struct sof_client_dev *cdev)
+{
+	int ret;
+
+	ret = sof_client_core_get(cdev);
+	if (ret)
+		return ret;
+
+	if (!try_module_get(cdev->auxdev.dev.driver->owner)) {
+		sof_client_core_put(cdev);
+		return -ENODEV;
+	}
+
+	return 0;
+}
+
+static void sof_probes_put_modules(struct sof_client_dev *cdev)
+{
+	module_put(cdev->auxdev.dev.driver->owner);
+	sof_client_core_put(cdev);
+}
+
```

Comment:
ok, so that's a bug in the compress framework - I don't see the point of doing this for probes when we know @dbaluta will be adding support for compressed data soonish.

Reply:
Found it and I have a pending patch:
https://github.com/ujfalusi/sof-linux/commit/c2f77b5b034a5d1a81d36aba3ad7a4139c7b640c